effectful/src/Effectful/Console/Static.hs

```diff
@@ -0,0 +1,29 @@
+-- | An effect for reading/writing to standard streams.
```

Comment:
The description is outdated since now the module is internal.

Making it similar to the other internal `.Effect` modules will do.

Reply:
Done.